examples/resourceServer/server/index.js

```diff
@@ -26,6 +27,6 @@ function initializeClients() {
     }
 }
 
-// on startup, update/create the allowed clients.
-Meteor.startup(initializeClients);
 
+// On startup, update/create the allowed clients.
+Meteor.startup(initializeClients);
```

Comment:
Based on a previous comment, the whole concept of initializing clients on startup is moot. The fate of this code is dependent on the ratification of the aforementioned discussion.